wagtail/snippets/templates/wagtailsnippets/bulk_actions/confirm_bulk_delete.html

```diff
@@ -3,7 +3,7 @@
 
 {% block titletag %}
     {% if items|length == 1 %}
-        {% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}Delete {{ snippet_type_name }}{% endblocktrans %} - {{ items|first }}
+        {% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}Delete {{ snippet_type_name }}{% endblocktrans %} - {{ items.0.item }}
```

Comment:
Found another bug where the context dictionary for the item is used instead, making the title unreadable. This change fixes it.